src/pages/ReportAvatar.tsx

```diff
@@ -54,4 +61,7 @@ export default withOnyx<ReportAvatarProps, ReportAvatarOnyxProps>({
     policies: {
         key: ONYXKEYS.COLLECTION.POLICY,
     },
+    groupChatDraft: {
+        key: ONYXKEYS.NEW_GROUP_CHAT_DRAFT,
+    },
```

Comment:
Let's make this use the `useOnyx` hook instead. It should help avoid loading the data from onyx if we are not going to use it. Related to the point above ^